packages/subnav/theme.module.css

```diff
@@ -0,0 +1,47 @@
+/*
+Theme settings.
+Consumed in nearly all CSS files in this component,
+including in /partials.
+*/
+
+.light {
+  /* main */
+  --background-color: var(--white);
+  --border-color: var(--gray-5);
+  --dropdown-background-color: var(--white);
+  --dropdown-box-shadow: 0 8px 12px rgba(37, 38, 45, 0.08);
+
+  /* title or logo link */
+  --title-text-color: var(--gray-1);
+  --logo-text-color: var(--black);
+
+  /* github-stars-link */
+  --gh-stars-icon-color: var(--black);
+  --gh-stars-scrim-color: rgba(0, 0, 0, 0.1);
+
+  /* menu items */
+  --menu-item-text-color: var(--black);
+  --menu-item-underline-color: var(--black);
+  --menu-item-heading-text-color: var(--gray-3);
+}
+
+.dark {
```

Comment:
I've riffed off the existing implementation at cloud.hashicorp.com. Have also changed the modals to match the dark theme of the bar to avoid a checkerboard look.

Not 100% sure if this matches design's intent - I found what I think is the [reference Figma file](https://www.figma.com/file/qy89mZUl1HHFZ1d3Pc4s29/HCP-.com-Experience-Landing-Page?node-id=2605%3A9358) but could not find deets on the nav theming. So, went with what I _think_ would match design intent.